src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --border-sol-col: 1px solid #e1e7ed;
+}
+
+.form {
+  font-family: "Avenir", sans-serif;
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  border-radius: 4px;
+  border: var(--border-sol-col);
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.form__search {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+}
+
+.search__logo--big {
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+}
+
+.search__logo--small {
+  width: 11px;
+  height: 11px;
+  margin-left: 13px;
+}
+
+.search__input {
+  width: 100%;
+  padding-left: 62px;
+  border-radius: 3px;
+  border: none;
+  height: 100%;
+  font-size: 16px;
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.search__input:hover {
+  border-radius: 3px;
+  border: var(--border-sol-col);
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  width: 100%;
+  outline: none;
+  border-radius: 3px;
+  border: var(--border-sol-col);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
Color #fff is duplicated, so make sense to move it to the variable 
Check other duplicated styles

Reply:
but #fff  its color for linear-gradient  in input. another background-color: #fff  its for Form

Reply:
I think It`ll be little mix up